File: atividade-a1/src/vertice.cc
#include "vertice.h"

int Vertice::getId() {
    return _id;
}

int Vertice::getGrau() {
    return _grau;
}

std::string Vertice::getRotulo() {
    return _rotulo;
}

ListaEncadeada<Vertice> Vertice::getVizinhos() {
    return _vizinhos;
}

File: atividade-a1/python/vertice.py
class Vertice:
    def __init__(self, id, rotulo):
        self.__id = id
        self.__rotulo = rotulo
        self.__vizinhos = {}
    
    @property
    def id(self):
        return self.__id
    
    @property
    def rotulo(self):
        return self.__rotulo
    
    @property
    def vizinhos(self):
        return self.__vizinhos

